### ISSUE TITLE
Show/hide Todoist shortcut not working

### ISSUE THREAD

**Issue by jurf** [flathub/com.todoist.Todoist#15]
There is no error anywhere or anything. No idea what could be wrong.

**Comment by jurf** [comment]
Well, it’s back, and it works to hide Todoist, but not to show it. It’s probably got to do with Wayland.

**Comment by raimue** [comment]
Configuring the keyboard shortcut for "Show / Hide Todoist" works fine for me with version 9.10.0, but I am still on Xorg. Has anyone reproduced this issue on Wayland with a recent version?

**Comment by gozes** [comment]
I've been trying to get this working on GNOME 48 on Fedora 42, which now has Wayland global keyboard support, but it does not seem to be working at all, regardless of what keymap I set it to.

**Comment by Franz1241** [comment]
It wont work on Wayland because even if Wayland global keyboard support is now available the Todoist team has to use the correct APIs on the app to call it. The workaround is to manually set a custom keyboard shortcut in the Settings page calling ` flatpak run com.todoist.Todoist` or if you installed in on AppImage the path to that

**Comment by gozes** [comment]
> It wont work on Wayland because even if Wayland global keyboard support is now available the Todoist team has to use the correct APIs on the app to call it. The workaround is to manually set a custom keyboard shortcut in the Settings page calling ` flatpak run com.todoist.Todoist` or if you installed in on AppImage the path to that

@Franz1241 but that will only bring focus to todoist not just bringing up the quick add box correct?

**Comment by Franz1241** [comment]
Yes, but once you get Todoist on focus you can use Todoist specific shortcuts like "Q" to pop up the quick add box
